assets/css/detours/_detour_table.scss

```diff
@@ -61,6 +61,12 @@ $table-border-radius: 0.5rem;
     td:last-child {
       border-bottom-right-radius: $table-border-radius;
     }
+
+    @include media-breakpoint-down(md) {
```

Comment:
This is an unrelated visual bug fix, but I noticed it while working on this PR and didn't realize until just now that I had accidentally lumped it into this PR.

I can split it out to a separate PR if you'd all prefer, but I didn't necessarily think it was worth the effort.